draft-ietf-gnap-core-protocol.md

```diff
@@ -6867,6 +6899,9 @@ Throughout many parts of GNAP, the parties pass shared references between each o
 
 > Note: To be removed by RFC editor before publication.
 
+- -14
+    - Fix key rotation with HTTP Signatures based on security analysis.
```

Comment:
Which security analysis are we referring to ?

Reply:
It was done on HTTPSig, thread starts here: https://lists.w3.org/Archives/Public/ietf-http-wg/2023JanMar/0062.html